src/libraries/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -25,7 +27,12 @@ internal static class UnixCommandLinePing
             string fileName = ipv4 ? s_ipv4PingFile : s_ipv6PingFile;
             foreach (string folder in s_binFolders)
             {
+                string pathv4v6 = Path.Combine(folder, s_ipv4v6PingFile);
                 string path = Path.Combine(folder, fileName);
+                if (File.Exists(pathv4v6))
```

Comment:
It seems like this will prefer 'ping` over 'ping6'. I don't know if it is safe

https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-103838-merge-5862a84e5eb54ce381/System.Net.Ping.Functional.Tests/3/console.6b3e6490.log?helixlogtype=result

Reply:
that should be the case. names ping4 and ping6 do not exist in iputils anymore.